### PR TITLE
add new Loki based alert rule

### PR DESCRIPTION
**Pull request by Abuelodelanada**
## Issue

Based on Traefik log format, this PR adds a Loki based alert rule that will fire if a `level=error` string is found.

The alert rule will work once PR #104 (or a PR that replace it) is finally merged.

For the time being, this file is harmless to the functionality of charm.

### CODE REVIEW

**Review by PietroPasotti — APPROVED** [review]

**Comment by PietroPasotti** [comment]
failing CI -> snappy outage

File: src/loki_alert_rules/error_level_logs_found.rule [hunk 1]
```diff
@@ -0,0 +1,7 @@
+alert: ErrorLevelLogsFound
+expr: rate({%%juju_topology%%} |= "level=error"[1m]) > 0
```
Comment (sed-i) [inline src/loki_alert_rules/error_level_logs_found.rule:2]: Shall we start using the `job~=.+` thingy instead of `%%juju_topology%%`?

**Review by sed-i — APPROVED** [review]

File: src/loki_alert_rules/error_level_logs_found.rule (hunk 1 above)
Comment (Abuelodelanada) [reply src/loki_alert_rules/error_level_logs_found.rule:2]: Not until we make a decision [on this](https://github.com/canonical/loki-k8s-operator/issues/229).

Comment (sed-i) [reply src/loki_alert_rules/error_level_logs_found.rule:2]: My impression was that it's independent: we don't need `%%juju_topology%%`, and with it it won't pass cos-tool validation, and with `job=~".+"` everything works, so might as well..?

**Pull request merged**